crashsimilarity/crash_similarity.py

```diff
@@ -14,39 +14,22 @@
 
 from crashsimilarity import utils
 from crashsimilarity.downloader import SocorroDownloader
+from crashsimilarity.utils import StackTraceProcessor
 
 pyximport.install()
 
 
-def clean_func(func):
-    func = func.lower().replace('\n', '')
-
-    if '@0x' in func:
-        return func[:func.index('@0x') + 3]
-
-    return func
-
-
-def preprocess(stack_trace):
-    return [clean_func(f) for f in stack_trace.split(' | ')][:10]  # XXX: 10 bottom frames or all of them?
-
-
-# Exclude stack traces without symbols.
-def should_skip(stack_trace):
-    return 'xul.dll@' in stack_trace or 'XUL@' in stack_trace or 'libxul.so@' in stack_trace
-
-
 def read_corpus(fnames):
```

Comment:
@KirillTim each PR/commit should be as clean as possible and as self-contained as possible, they're also easier to review this way. I know we are refactoring, but we should keep a clean history.
Using the `process` function here is a really simple change to do, so it's worth doing it. If it was a hard and big change I would agree with you.